monkey/monkey_island/cc/resources/local_run.py

```diff
@@ -43,7 +43,29 @@ def run_local_monkey():
 
     # run the monkey
     try:
-        args = ['"%s" m0nk3y -s %s:%s' % (target_path, local_ip_addresses()[0], env.get_island_port())]
```

Comment:
Why do we need to pass the whole list of servers to monkey here? If monkey is running on local machine, it will be able to connect back to any local interface, no?